src/V3DfgAstToDfg.cpp

```diff
@@ -79,6 +79,18 @@ class AstToDfgVisitor final : public VNVisitor {
     // AstNode::user1p   // DfgVertex for this AstNode
     const VNUser1InUse m_user1InUse;
 
+    // TYPES
+    // Represents a driver during canonicalization
+    struct Driver {
+        FileLine* flp;
+        uint32_t lsb;
+        DfgVertex* vtxp;
```

Comment:
```suggestion
        FileLine* m_fileline;
        DfgVertex* m_vtxp;
        uint32_t m_lsb;
```

And comment.  The reordering is to save 4 bytes.  (I realize you just moved code, I missed this earlier)

Reply:
FWIW will do this, but it does not save anything as the size of the struct needs to be a multiple of its alignment and its alignment is the largest of its members.